documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+  - that the client attests the information prepared by the issuer in the delivery channel to be true
+  - that the client successfully got the signed information in the format it wants by denoting the need as solved
+4) private delivery channel of issuer, destroyed after solving the need or at max for X minutes or service shutdown whatever comes first
+  - that a service with some id is attending to a single specific need that has been noticed for what : (ssid, attendTo, what)
+  - all the information the issuer has prepared to issue, taken from the NLX result channel
+  - that the specific client in need is allowed to access this channel
+  - that it matches a specific need for that same what
+  - that it attests (with signature using NLX outway key) the information to be issued
+5) clerk / auth svc channel, held in memory until read or at max for X minutes or service shutdown whatever comes first
+  - that the client has been identified / authenticated (and authorised) to retrieve the product in relation to the BSN (or other source argument configured) denoted in a given BSN claim in the private customer channel
+6) private NLX channel of issuer, held in memory until read or at max for X minutes or service shutdown whatever comes first
+    - the result of a NLX call stored in the channel of and thus signed with the NLX certificate of the issuer (every claim specifically allowed to specific other actors)
+7) validation channel of validator, destroyed after validation or service shutdown whatever comes first
+      - the same claim and attestation pair as was issued from the delivery channel of the issuer, read from the QR code on a piece of paper
+
+besides these channels, information is issued and stored in supported verifiable credential platforms (in privately held wallets of the client as holder). For now only a custom paper wallet platform is used which stores an attested claim in a QR code printed on a piece of paper to be owned by the client. This piece of paper shall also contain the raw information contained in the attested claim in the QR code which limits the amount of data that can be held in claims such you can issue them in a paper wallet.
+
+The solution just presents, signs and issues information as returned as result from the NLX call; for now it does not semantically parses this information. For now, the webservice called through NLX needs to accept a BSN number as argument that identifies the client though these kinds of arguments are configurable. The object of what is needed or attended to is something configured by the issuer like 'Uittreksel-GBA-Haarlem'. All other data results from usage of the abundance service api and discipl core api and the connectors used (ephemeral and NLX for now).
+
+The process for issuing proofs as described above is illustrated in verifiable claim sequence diagrams below:
+
+![](images/vcseq1.png)
+
+![](images/vcseq2.png)
+
+
+![](images/vcseq3.png)
```

Comment:
Allow from SSID S-X is before the refer in implementation

Reply:
seems to be already the case now in the latest diagrams